### DISCUSSION TITLE
Uninstall not working properly

### DISCUSSION THREAD

**Discussion by theking2** [valinet/ExplorerPatcher#1390] [Q&A]
Running 
```ps
.\ep_setup.exe /uninstall
```
results in this error: 

![image](https://user-images.githubusercontent.com/1612152/234889458-a3892218-5045-4ddf-86e8-eaff79806687.png)

and appears not to uninstall the application. None of the four suggestions seems to solve this.

**Comment by tallman69** [comment]
What other uninstall methods have you tried?

**Comment by theking2** [comment]
`ep_setup.exe /uninstall`

**Reply by tallman69** [reply]
Have you tried a third party uninstaller such as Revo Uninstaller or IObit Uninstaller?  If you install IObit Uninstaller be careful when you install it because it will want to install other unnecessary programs along with it, but the program works well.

**Comment by pyrates999** [comment]
Did you try reinstalling and then uninstalling?

**Reply by theking2** [reply]
that might've worked as it is gone now
